codecov.yml

```diff
@@ -0,0 +1,2 @@
+ignore:
+  - Tests
```

Comment:
This still doesn't seem to be ignoring the `Tests` folder.

Maybe try `"Tests"` or `"Tests/**/*"`